examples/expectations/query_expectation_template.py

```diff
@@ -0,0 +1,117 @@
+"""
+This is a template for creating custom QueryExpectations.
+For detailed instructions on how to use it, please see:
+    https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_query_expectations
+"""
+
+from typing import Any, Dict, Optional, Union
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.exceptions.exceptions import (
+    InvalidExpectationConfigurationError,
+)
+from great_expectations.execution_engine import ExecutionEngine
+from great_expectations.expectations.expectation import (
+    ExpectationValidationResult,
+    QueryExpectation,
+)
+
+
+# This class defines the Expectation itself
+# <snippet>
```

Comment:
Do we still need these `<snippet>`s in the examples directory? Shouldn't this also be in `docs/integration`? Like with the others, we can have something in both places, but having this clutter in the examples is probably not preferred.

Reply:
Seems reasonable to me!